frontend/src/global_styles/openproject/_primer-adjustments.sass

```diff
@@ -49,7 +49,7 @@ ul.tabnav-tabs
   margin-left: 0
 
 /* Remove margin-left: 2rem from Breadcrumbs */
-#breadcrumb
+#breadcrumb,page-header
```

Comment:
A bit further down in that file there is that block 

```
/* TODO: The actions within the PageHeader are currently not aligned correctly.
   The pageHeader itself already uses center alignment but the actions themselves default to 'normal'.
   Will be added to our primer repository */
.PageHeader-actions
  align-items: center
```

Could you please remove that, as it is now part of the PageHeader and thus not needed any more